src/helpers/sparkPostEmail.ts

```diff
@@ -19,36 +19,32 @@ let payload = {
     "recipients": []
 };
 
-const sendEmail = (user, otp, recipientEmail) => {
+const sendEmail = async (user: string, otp: string, recipientEmail: string): Promise<any> => {
```

Comment:
No `any`

Reply:
Need to define the exact type for `Promise<any>`

Reply:
I have updated any.
I don't no 'ANY' anymore. :+1: